### PR TITLE
Create ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61355](https://cumulocity.atlassian.net/browse/MTM-61355)
Original PR: [7716](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7716)

[MTM-61355]: https://cumulocity.atlassian.net/browse/MTM-61355?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: added operationsCreated and operationUpdated columns for usageStatistics (#7688) [GRAFT][release/cd] (#7716)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md]: The Claude AI suggested to use the following title:
```suggestion
title: Add operationsCreated and operationsUpdated columns to usageStatistics
```

File: content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61355
+version: 1021.16.0
+---
+added operationsCreated and operationUpdated columns for usageStatistics (#7688) [GRAFT][release/cd] (#7716)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md]: The Claude AI suggested to use the following body:
```suggestion
To provide more detailed insights into platform usage, the usageStatistics database table has been extended with two new columns: operationsCreated and operationsUpdated. These columns track the number of operations that have been created and updated, respectively. This change enhances the granularity of the usage statistics, enabling better analysis and understanding of platform activity. Existing usage statistics data and functionality remain unaffected by this addition.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@darlucumulo Please review and approve

File: content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md (hunk 2 above)
Comment (jkot-c8y) [reply content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md]: rewritten suggestion (possible to add links to Rest API release notes regarding backend)
```suggestion
To provide more detailed insights into platform usage, the Usage Statistics UI view for {{< enterprise-tenant >}}s has been extended with two new columns: `operationsCreated` and `operationsUpdated`. These columns present the number of operations that have been created and updated, respectively.

This change enhances the granularity of the usage statistics request counters, enabling better analysis and understanding of platform activity. Existing usage statistics data and functionality remain unaffected by this addition. 

New UI feature is compatible with backend version **10.20.611.0**, which introduced new fields in REST API. This feature works predictably with backend version **10.20.664.0** and later, where all operations creations and updates by users and devices are tracked by {{< product-c8y-iot >}} request counters.
```

File: content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md (hunk 1 above)
Comment (jkot-c8y) [reply content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md]: rewritten suggestion
```suggestion
title: Added operationsCreated and operationsUpdated columns to usageStatistics view
```

File: content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61355
+version: 1021.16.0
+---
+To provide more detailed insights into platform usage, the Usage Statistics UI view for {{< enterprise-tenant >}}s has been extended with two new columns: `operationsCreated` and `operationsUpdated`. These columns present the number of operations that have been created and updated, respectively.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md]:
```suggestion
To provide more detailed insights into platform usage, the **Usage statistics** page for {{< enterprise-tenant >}}s has been extended with two new columns: "Operations Created" and "Operations Updated". These columns present the number of operations that have been created and updated, respectively.
```

File: content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+date: ""
+title: Added operationsCreated and operationsUpdated columns to usageStatistics view
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-16-0-added-operations-created-and-operation-updated-columns-for-usage-statistics-7688.md]:
```suggestion
title: Added two new columns to the Usage Statistics page
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@JanePorter @martin-c8y Please approve

**Review by jkot-c8y — APPROVED** [review]

**Review by martin-c8y — APPROVED** [review]

**Pull request merged**